migrations/20220324080020-add-place-comment.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.createTable(
+        'comments',
+        {
+          id: {
+            type: Sequelize.UUID,
+            primaryKey: true,
+            allowNull: false,
+            unique: true,
+            autoIncrement: false,
+            defaultValue: Sequelize.fn('uuid_generate_v4'),
+          },
+          title: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          message: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          links: {
+            type: Sequelize.ARRAY(Sequelize.TEXT),
+            allowNull: true,
+          },
```

Comment:
Can we set it to empty array by default? Or is Sequelize handling that (when null then returns [])?

Reply:
👍